diff --git a/node_modules/react-native-double-tap/index.js b/node_modules/react-native-double-tap/index.js
index 812a47e..431a29f 100644
--- a/node_modules/react-native-double-tap/index.js
+++ b/node_modules/react-native-double-tap/index.js
@@ -1,5 +1,5 @@
 import React, { Component } from "react";
-import { TouchableOpacity } from "react-native";
+import { Pressable, TouchableOpacity, TouchableWithoutFeedback } from "react-native";
 
 export default class DoubleClick extends Component {
   constructor(props) {
@@ -51,9 +51,9 @@ export default class DoubleClick extends Component {
 
   render() {
     return (
-      <TouchableOpacity onPress={this._onPress}>
+      <Pressable onPress={this._onPress}>
         {this.props.children}
-      </TouchableOpacity>
+      </Pressable>
     );
   }
 
